Rodents/Portal_rodent.csv

```diff
@@ -66467,3 +66467,102 @@
+66615,4,1,2017,-460,13,8,11,"PP","M",,,"S",,,,,22,23,"B20400",,,,,,,,,
+66616,4,1,2017,-460,13,8,66,"PP","M",,,"S",,,,,20,22,"PP5",,,,,,,,,
+66617,4,1,2017,-460,13,8,56,"PE",,,,,,,,,,,"B21146",,,,,,,,,
+66618,4,1,2017,-460,13,8,65,"DM","M",,,"S",,,,,36,47,"B2C50A",,,,,,,,,
+66619,4,1,2017,-460,13,8,61,"PP","M",,,"S",,,,,21,20,"B73E65",,,,,,,,,
+66620,4,1,2017,-460,14,8,41,"DO","F",,,,,,"E",,35,60,"B2C686",,,,,,,,,
+66621,4,1,2017,-460,14,8,56,"DM","M","Z",,,,,,,37,44,"B2D1BD",,,,,,,,,
+66622,4,1,2017,-460,14,8,54,"OT",,,,,,,,,,,"B715C8",,,,,,,,,
+66623,4,1,2017,-460,14,8,52,"PP","F",,,,,,"E",,21,18,"A51565",,,,,,,,,
+66624,4,1,2017,-460,14,8,51,"DO","M",,,"S",,,,,36,53,"B7144D",,,,,,,,,
+66625,4,1,2017,-460,14,8,33,"DO","M",,,"S",,,,,35,52,"DO6",,,,,,,,,
+66626,4,1,2017,-460,14,8,74,"DO","M",,,"S",,,,,35,,"B19BB1",,,,,,,,,
+66627,4,1,2017,-460,14,8,77,"OT",,,,,,,,,,,"B2B344",,,,,,,,,
+66628,4,1,2017,-460,14,8,14,"DO","M",,,"S",,,,,33,49,"DO7",,,,,,,,,
+66629,4,1,2017,-460,14,8,15,"PM",,,,,,,,,,,,,,,,,,,,
+66630,4,1,2017,-460,14,8,37,"DM","F",,,,,,"E",,36,52,"B20652",,,,,,,,,
+66631,4,1,2017,-460,14,8,72,"DO","F",,,,"S",,,,36,,"B1DB12",,,,,,,,,
+66632,4,1,2017,-460,14,8,65,"AH",,,,,,,,,,,,,,,,,,,,
+66633,4,1,2017,-460,14,8,21,"DM","M",,,"S",,,,,37,48,"B2CEEE",,,,,,,,,
+66634,4,1,2017,-460,14,8,17,"PE",,,,,,,,,,,"B1F3EE",,,,,,,,,
+66635,4,1,2017,-460,17,8,57,"PP","M",,,"R",,,,,21,22,"B1F2EB",,,,,,,,,
+66636,4,1,2017,-460,17,8,47,"PP","M",,,"S",,,,,20,21,"B2C9EC",,,,,,,,,
+66637,4,1,2017,-460,17,8,54,"DM","F",,,,,,"E",,34,49,"B212CD",,,,,,,,,
+66638,4,1,2017,-460,17,8,41,"DM","M",,,"S",,,,,36,42,"B207F3",,,,,,,,,
+66639,4,1,2017,-460,17,8,16,"DM","M",,,"R",,,,,35,43,"B20631",,,,,,,,,
+66640,4,1,2017,-460,17,8,75,"DM","F",,,,,,"R",,36,48,"B6E945",,,,,,,,,
+66641,4,1,2017,-460,17,8,74,"PP","M",,,"S",,,,,22,22,"B21151",,,,,,,,,
+66642,4,1,2017,-460,17,8,11,"DM","F",,,,,,"R",,34,39,"B2CE4E",,,,,,,,,
+66643,4,1,2017,-460,18,8,42,"DM","F","Z",,,,,,,36,35,"B1F0CF",,,,,,,,,
+66644,4,1,2017,-460,18,8,73,"DM","F",,,,"S",,,,35,29,"B25194",,,,,,,,,
+66645,4,1,2017,-460,18,8,15,"DM","F",,,,,,"E",,36,60,"B1DC8F",,,,,,,,,
+66646,4,1,2017,-460,18,8,37,"NA",,,,,,,,,,,"B2CE1C",,,,,,,,,
+66647,4,1,2017,-460,18,8,46,"SH",,,,,,,,,,,"B1F305",,,,,,,,,
+66648,4,1,2017,-460,18,8,15,"SS",,,,,,,,,,,,,,,,,,,,
+66649,4,1,2017,-460,18,8,76,"PE",,,,,,,,,,,,,,,,,,,,
+66650,4,1,2017,-460,18,8,55,"PE",,,,,,,,,,,"B1FDDD",,,,,,,,,
+66651,4,1,2017,-460,18,8,27,"RM",,,,,,,,,,,"B2C924",,,,,,,,,
+66652,4,1,2017,-460,9,8,,,,,,,,,,,,,"B2C564",,,,,,,,,"R"
+66653,4,1,2017,-460,12,8,,,,,,,,,,,,,"B2C4E5",,,,,,,,,"R"
+66654,4,1,2017,-460,12,8,,,,,,,,,,,,,"B71B77",,,,,,,,,"R"
+66655,4,1,2017,-460,9,8,,,,,,,,,,,,,"B2D109",,,,,,,,,"R"
+66656,4,1,2017,-460,9,8,,,,,,,,,,,,,"B20072",,,,,,,,,"R"
+66657,4,1,2017,-460,9,8,,,,,,,,,,,,,"B2D290",,,,,,,,,"R"
+66658,4,1,2017,-460,10,8,,,,,,,,,,,,,,,,,,,,,,
+66659,4,1,2017,-460,16,8,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Yeah, I mean, without those numbers it's a bit misleading, since it looks like the number of traps you did record were the actual numbers trapped on those plots. But you know, it's not a huge deal for a non-census period. At least if you have a row that looks like `10, 8, UR` then it's obvious that the plot was trapped and had (>0) rodents.